src/destreamer.ts

```diff
@@ -1,90 +1,34 @@
-import { sleep, parseVideoUrls, checkRequirements, makeUniqueTitle, ffmpegTimemarkToChunk } from './utils';
+import {
+    sleep, parseVideoUrls, checkRequirements, makeUniqueTitle,
+    ffmpegTimemarkToChunk, makeOutputDirectories, getOutputDirectoriesList, checkOutDirsUrlsMismatch
+} from './utils';
+import { getPuppeteerChromiumPath } from './PuppeteerHelper';
+import { setProcessEvents } from './Events';
+import { ERROR_CODE } from './Errors';
 import { TokenCache } from './TokenCache';
 import { getVideoMetadata } from './Metadata';
-import { Metadata, Session, Errors } from './Types';
+import { Metadata, Session } from './Types';
 import { drawThumbnail } from './Thumbnail';
+import { argv } from './CommandLineParser';
 
 import isElevated from 'is-elevated';
 import puppeteer from 'puppeteer';
 import colors from 'colors';
 import fs from 'fs';
```

Comment:
Allow me to earn my money here and become snobulinter -
`'fs' is declared but its value is never read.` :)